taxonomies/categories.txt

```diff
@@ -6799,777 +6799,6 @@ nl:Natuurlijke mineraalwaters, Natuurlijk mineraalwater
-agribalyse_food_code:en:76024
-ciqual_food_code:en:76024
-ciqual_food_name:en:Mineral sparkling water (Vernière), bottled, averagely mineralized
-ciqual_food_name:fr:Eau minérale Vernière, embouteillée, gazeuse, moyennement minéralisée (Les Aires, 34)
-
-<en:Carbonated waters
-<en:Natural mineral waters
-en:Bottled strongly mineralized sparkling water Didier
-fr:Eau minérale embouteillée gazeuse fortement minéralisée Didier Martinique
-agribalyse_food_code:en:76018
-ciqual_food_code:en:76018
-ciqual_food_name:en:Mineral sparkling water (Didier), bottled, strongly mineralized
-ciqual_food_name:fr:Eau minérale Didier, embouteillée, gazeuse, fortement minéralisée (Martinique)
-
-<en:Carbonated waters
-<en:Natural mineral waters
-en:Bottled strongly mineralized sparkling water Chateauneuf
-fr:Eau minérale embouteillée gazeuse fortement minéralisée Chateauneuf 63
-agribalyse_food_code:en:76013
-ciqual_food_code:en:76013
-ciqual_food_name:en:Mineral sparkling water (Chateauneuf), bottled, strongly mineralized
-ciqual_food_name:fr:Eau minérale Chateauneuf, embouteillée, gazeuse, fortement minéralisée (Chateauneuf, 63)
-
-<en:Carbonated waters
-<en:Natural mineral waters
-en:Bottled strongly mineralized sparkling water Chateldon
-fr:Eau minérale embouteillée gazeuse fortement minéralisée Chateldon 63
-agribalyse_food_code:en:76014
-ciqual_food_code:en:76014
-ciqual_food_name:en:Mineral sparkling water (Chateldon), bottled, strongly mineralized
-ciqual_food_name:fr:Eau minérale Chateldon, embouteillée, gazeuse, fortement minéralisée (Chateldon, 63)
-
-<en:Carbonated waters
-<en:Natural mineral waters
-en:Bottled averagely mineralized still water Clos de l'Abbaye
-fr:Eau minérale embouteillée non gazeuse moyennement minéralisée Clos de l'Abbaye St-Amand 59
-agribalyse_food_code:en:76015
-ciqual_food_code:en:76015
-ciqual_food_name:en:Mineral still water (Clos de l'Abbaye), bottled, averagely mineralized
-ciqual_food_name:fr:Eau minérale Clos de l'Abbaye, embouteillée, non gazeuse, moyennement minéralisée (St-Amand, 59)
-
-<en:Carbonated waters
-<en:Natural mineral waters
-en:Carbonated natural mineral waters
-de:natürliches Mineralwasser mit Kohlensäure versetzt, natürliche Mineralwässer mit Kohlensäure versetzt
-es:Aguas minerales naturales carbonatadas.
-fi:hiilihapotetut luontaiset kivennäisvedet
-fr:Eau minérale naturelle gazéifiée
-it:Acque minerali naturali carbonate
-nl:Met koolzuur versterkte mineraalwaters, Met koolzuur versterkt mineraalwater
-
-<en:Carbonated natural mineral waters
-en:Mildly carbonated natural mineral water
-de:natürliches Mineralwasser mit wenig Kohlensäure versetzt, natürliche Mineralwässer mit wenig Kohlensäure versetzt
-fi:kevyesti hiilihapotettu luonnollinen kivennäisvesi
-fr:Eau minérale naturelle légèrement gazéifiée
-it:Acqua minerale naturale mediamente carbonata
-nl:Met weinig koolzuur versterkte mineraalwaters, Met weinig koolzuur versterkt mineraalwater
-
```

Comment:
are the last 2 deletions intentional? they're not brands.

Reply:
good  catch, I'll put them back